tests/endpoints/events/biosero/test_destination_failed.py

```diff
@@ -25,7 +25,7 @@ def test_post_destination_completed_missing_barcode(app, client, biosero_auth_he
             "/events",
             data={
                 "user_id": "user1",
-                "event_type": "lh_biosero_cp_destination_failed",
+                "event_type": "lh_biosero_cp_destination_plate_failed",
```

Comment:
Could use the constant `EVENT_DESTINATION_FAILED` throughout in these tests

Reply:
changed all